daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -1199,37 +1259,26 @@ case class ParserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
     VerifyTestCase.verifyParserTestData(resultXmlNode, testInfoset, implString)
 
     (shouldValidate, expectsValidationError) match {
-      case (_, true) => {
-        // Note that even if shouldValidate is false, we still need to check
-        // for validation diagnostics because failed assertions with
-        // failureType="recoverableError" are treated as validation errors
-        VerifyTestCase.verifyAllDiagnosticsFound(
-          actual.getDiagnostics,
-          optExpectedValidationErrors,
-          implString
-        ) // verify all validation errors were found
-        Assert.invariant(actual.isValidationError)
-      }
       case (true, false) => {
```

Comment:
Do we need this case anymore? You've updated checkDiagnosicts to check validation errors, including whether or not to ignore unexpected errors. I think this match case goes away completely and we just use the default case? And verifyNoValidatioNErrorsFound goes away?